hubblestack/extmods/returners/slack_pulsar_returner.py

```diff
@@ -62,18 +62,37 @@
 import urllib
 
 # pylint: disable=import-error,no-name-in-module,redefined-builtin
-from salt.ext.six.moves.urllib.parse import urljoin as _urljoin  # pylint: disable=import-error,no-name-in-module
+from salt.ext.six.moves.urllib.parse import urljoin as _urljoin
+# pylint: disable=import-error,no-name-in-module
 import salt.ext.six.moves.http_client
 # pylint: enable=import-error,no-name-in-module,redefined-builtin
 
 # Import Salt Libs
 import salt.returners
 
-log = logging.getLogger(__name__)
```

Comment:
could we keep this one as `log` (rather than `LOG`) also please?

Reply:
yeah, I was planning to go over all the LOG changes and get them back to log, thanks for the heads up

Reply:
This was just the first one I saw...